modules/dhcp_native_ms/dhcp_native_ms_main.rb

```diff
@@ -95,11 +95,11 @@ def unused_ip(subnet_address, mac_address, from_address, to_address)
         foreman_end_addr_ip = IPAddr.new(to_address)
         dhcp_start_addr_ip = IPAddr.new(dhcp_start_addr)
         dhcp_end_addr_ip = IPAddr.new(dhcp_end_addr)
-        if not (dhcp_start_addr_ip..dhcp_end_addr_ip).cover? foreman_start_addr_ip
+        if !(dhcp_start_addr_ip..dhcp_end_addr_ip).cover? foreman_start_addr_ip
```

Comment:
Logically this is equivalent to `unless` as well, right? Same be low

Reply:
Style/NegatedIf does fix this